Makefile

```diff
@@ -23,11 +23,27 @@ clean:
 	rm -f node_modules/.uptodate .pydeps
 	rm -rf build
 
+.PHONY: env
+env:
+	@bin/hypothesis authclient add --name openstax --authority openstax.org --type confidential | tail -n 2 | sed -e 's/Client ID: /HYPOTHESIS_CLIENT_ID=/' | sed -e 's/Client Secret: /HYPOTHESIS_CLIENT_SECRET=/'
+
 ## Run the development H server locally
 .PHONY: dev
-dev: build/manifest.json .pydeps
-	@bin/hypothesis --dev init
-	@bin/hypothesis devserver
+dev:
+	@echo "initializing hypothesis..."
+	@bin/hypothesis init
+
+ifneq ($(add_credentials_to),)
```

Comment:
I cant say i've seen `add_credentials_to` before. How is this working?

Reply:
i'm expecting that to be passed in via command line ala https://github.com/openstax/tutor-server/pull/1756/files#diff-4e5e90c6228fd48698d074241c2ba760R32

i could for sure update the help section for both of these commands